ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+set -x
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
+base_domain=origin-ci-int-aws.dev.rhcloud.com
+cluster_domain="${cluster_name}.${base_domain}"
+
+export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred 
+
+# Load array created in setup-vips:
+# 0: API
+# 1: Ingress
+# 2: DNS
+declare -a vips
+mapfile -t vips < "${SHARED_DIR}"/vips.txt
+
+hosted_zone_id="$(aws route53 list-hosted-zones-by-name \
+            --dns-name "${base_domain}" \
+            --query "HostedZones[? Config.PrivateZone != \`true\` && Name == \`${base_domain}.\`].Id" \
+            --output text)"
+
+
+echo "Creating DNS records..."
+cat > "${SHARED_DIR}"/dns-create.json <<EOF
+{
+"Comment": "Create public OpenShift DNS records for VSphere IPI CI install",
+"Changes": [{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api.$cluster_domain.",
+      "Type": "A",
+      "TTL": 60,
+      "ResourceRecords": [{"Value": "${vips[0]}"}]
+      }
+    },{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api-int.$cluster_domain.",
```

Comment:
@patrickdillon I am not sure we need api-int pretty sure its created within the coredns instance with in-cluster services.